sympy/simplify/simplify.py

```diff
@@ -33,6 +33,38 @@
 import mpmath
 
 
+def separable(expr, symbols=[]):
```

Comment:
Use None or an empty tuple instead of `[]` as a default value. Mutable defaults are problematic in Python. There is no problem here but mutable defaults are suspicious so don't use them.